web/portal/src/main/webapp/groovy/portal/webui/container/UITableColumnContainer.gtmpl

```diff
@@ -13,7 +13,7 @@
   String uiComponentClass = uicomponent.getCssClass() == null ? "" : uicomponent.getCssClass();
 %>
 <div
-  class="UIContainer UITableColumnContainer <%=uiComponentClass%>  id="${uicomponent.id}">
+  class="UIContainer UITableColumnContainer <%=uiComponentClass%>" id="${uicomponent.id}">

```

Comment:
It's not the only `gtmpl` file in fact, I 'll take care of updating the others as well